### PR TITLE
Adding a few slides to While

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/course_materials/control_structures.html [hunk 1]
```diff
@@ -226,9 +232,71 @@ <h3><b>Simple While Loop</b></h3>
                         <li>When it's going to stop?!</li>
                     </ul>
                 </section>
-                <section>
-                    <h3><b>Break and Continue</b></h3>
-                    <pre><code data-trim data-noescape class="hljs lang-python">
+
+
+
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:237]: remove those redundant lines

File: docs/course_materials/control_structures.html [hunk 2]
```diff
@@ -226,9 +232,71 @@ <h3><b>Simple While Loop</b></h3>
...
+              <section>
+                <h3><b>While with condition</b></h3>
+                <ul>
+                  <li>step must be initialized first</li>
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:241]: step > A step

File: docs/course_materials/control_structures.html [hunk 3]
```diff
@@ -226,9 +232,71 @@ <h3><b>Simple While Loop</b></h3>
...
+                    </code></pre>
+
+                <ul>
+                  <li>We can run the same but backwards </li>
+                  <li> i -= 1 is the same as writing i = i -1 </li>
+                </ul>
+                <pre><code data-trim data-noescape class="hljs lang-python">
+i = 10
+while i > 0:
+  print(i)
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:259]: please use tab

File: docs/course_materials/control_structures.html [hunk 4]
```diff
@@ -226,9 +232,71 @@ <h3><b>Simple While Loop</b></h3>
...
+  i -= 1
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:260]: same

File: docs/course_materials/control_structures.html [hunk 5]
```diff
@@ -226,9 +232,71 @@ <h3><b>Simple While Loop</b></h3>
...
+                <ul>
+                  <li>With the break statement we can stop the loop even if the while condition is true</li>
+                </ul>
+                <pre><code data-trim data-noescape class="hljs lang-python">
+i = 1
+while i < 10:
+  print(i)
+  if i == 3:
+    break
+  i += 1
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:275]: same - tabs

File: docs/course_materials/control_structures.html [hunk 6]
```diff
@@ -226,9 +232,71 @@ <h3><b>Simple While Loop</b></h3>
...
+                <ul>
+                  <li>Continue to the next iteration if i is 3</li>
+                </ul>
+                <pre><code data-trim data-noescape class="hljs lang-python">
+i = 0
+while i < 10:
+  i += 1
+  if i == 3:
+    continue
+  print(i)
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:289]: same

File: docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md [hunk 7]
```diff
@@ -10,10 +10,9 @@ Before answering the questions:
 
 Create a file *variables_1.py*
 
-1. Declare three variables (**x**,**y**, and **z**) as integers.
-2. Assign a value of 9 to **x**
-3. Assign a value of 7 to **y**
-4. Make **z** equal to the sum of those two variables. 
+2. Declare a variable **x** with the value 9 to
```
Comment (sleviim) [inline docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md:13]: remove the to at the end

File: docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md [hunk 8]
```diff
@@ -10,10 +10,9 @@ Before answering the questions:
 
 Create a file *variables_1.py*
 
-1. Declare three variables (**x**,**y**, and **z**) as integers.
-2. Assign a value of 9 to **x**
-3. Assign a value of 7 to **y**
-4. Make **z** equal to the sum of those two variables. 
+2. Declare a variable **x** with the value 9 to 
+3. Declare a variable **x** with the value 7 to
```
Comment (sleviim) [inline docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md:14]: same

File: docs/course_materials/exercises/04_Compound_data_types/homework_dictionary.md [hunk 9]
```diff
@@ -7,7 +7,7 @@
 
 Create a file *countries.py*
 
-- Create a new dictionary with the Udi number of flights to each country:  “Italy”:2 “Spain”:3  “Israel”:1
+- Create a new dictionary with Udi's flights number to each country:  “Italy”:2 “Spain”:3  “Israel”:1
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/homework_dictionary.md:10]: split into 2 lines:
Create a new dictionary with Udi's flights' number to each country: <br>
“Italy”:2 “Spain”:3  “Israel”:1

File: docs/course_materials/exercises/05_Control_structures/class_exercise_while_1.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+### Let's Practice Together #2
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/class_exercise_while_1.md:1]: add \ before the #

File: docs/course_materials/exercises/05_Control_structures/class_exercise_if.md [hunk 11]
```diff
@@ -0,0 +1,5 @@
+### Let's Practice Together \#1
+
+- Create a file *is_positive.py*
+- In this file, write a program that defines parameter *a*.
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/class_exercise_if.md:4]: *a* > **a**

**Review by sleviim — CHANGES_REQUESTED** [review]

File: docs/course_materials/control_structures.html (hunk 2 above)
Comment (shiramax) [reply docs/course_materials/control_structures.html:241]: it's step  variable

Comment (sleviim) [reply docs/course_materials/control_structures.html]: So maybe "The step variable"

**Review by sleviim — APPROVED** [review]

**Pull request merged**